terracumber_config/tf_files/SUSEManager-4.1-NUE.tf

```diff
@@ -81,8 +81,8 @@ provider "libvirt" {
 module "cucumber_testsuite" {
   source = "./modules/cucumber_testsuite"
 
-  product_version = "4.1-nightly"
-  
+  product_version = "4.1-released"
```

Comment:
I don't really understand why this PR is changing this. The two TFs are used to test Devel:Galaxy:Manager:4.1*

If you change it to released, you will test the released code.